vm/src/stdlib/posix.rs

```diff
@@ -413,6 +413,63 @@ pub mod module {
         )
     }
 
+    #[derive(FromArgs)]
+    struct MknodArgs {
+        #[pyarg(any)]
+        path: PyPathLike,
+        #[pyarg(any)]
+        mode: libc::mode_t,
+        #[pyarg(any)]
+        device: libc::dev_t,
+        #[pyarg(flatten)]
+        dir_fd: DirFd<1>,
```

Comment:
here is a warning. please check the clippy result. `#[allow]` looks like to be needed